gc/base/MemorySubSpace.cpp

```diff
@@ -2010,6 +2018,12 @@ MM_MemorySubSpace::wasContractedThisGC(uintptr_t gcCount)
 
 uintptr_t
 MM_MemorySubSpace::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
```

Comment:
Since this is the base class, and subclasses will override only one of the 2 variants, we indeed need default implementation for both of these methods. But I believe both of them should really just assert unreachable.
Specifically, 
- Generational subclass will only need the variant with parameters and never use one without
- Flat/SemiSpace/Generic will only need the variant without parameters and never use the one with

Reply:
At some point when we start using checkpoint for GC policies other than standard ones, we'll  have to touch some more Subspaces (Tarok, Segregated,...)